src/test/java/racingGameTest/RacingWinnerTest.java

```diff
@@ -0,0 +1,33 @@
+package racingGameTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import racingGame.domain.Car;
+import racingGame.domain.Winner;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingWinnerTest {
+    List<Car> racingCars;
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/java-racingcar/pull/645#discussion_r399752563

```suggestion
    private List<Car> racingCars;
```